data/data-pipeline/data_pipeline/tests/score/test_tiles_smoketests.py

```diff
@@ -0,0 +1,195 @@
+# flake8: noqa: W0613,W0611,F811
+from dataclasses import dataclass
+from typing import Optional
+import pandas as pd
+import numpy as np
+import pytest
+from data_pipeline.config import settings
+from data_pipeline.etl.score import constants
+from data_pipeline.score import field_names
+from data_pipeline.etl.score.constants import (
+    TILES_SCORE_COLUMNS,
+    THRESHOLD_COUNT_TO_SHOW_FIELD_NAME,
+    USER_INTERFACE_EXPERIENCE_FIELD_NAME,
+)
+from .fixtures import final_score_df  # pylint: disable=unused-import
+
+pytestmark = pytest.mark.smoketest
+
+
+@pytest.fixture
+def tiles_df(scope="session"):
+    return pd.read_csv(
+        settings.APP_ROOT / "data" / "score" / "csv" / "tiles" / "usa.csv",
+        dtype={"GTF": str},
+        low_memory=False,
+    )
+
+
+PERCENTILE_FIELDS = [
+    "DF_PFS",
+    "AF_PFS",
+    "HDF_PFS",
+    "DSF_PFS",
+    "EBF_PFS",
+    "EALR_PFS",
+    "EBLR_PFS",
+    "EPLR_PFS",
+    "HBF_PFS",
+    "LLEF_PFS",
+    "LIF_PFS",
+    "LMI_PFS",
+    "MHVF_PFS",
+    "PM25F_PFS",
+    "P100_PFS",
+    "P200_I_PFS",
+    "P200_PFS",
+    "LPF_PFS",
+    "KP_PFS",
+    "NPL_PFS",
+    "RMP_PFS",
+    "TSDF_PFS",
+    "TF_PFS",
+    "UF_PFS",
+    "WF_PFS",
+    "UST_PFS",
+]
+
+
+def test_percentiles(tiles_df):
+    for col in PERCENTILE_FIELDS:
+        assert tiles_df[col].min() >= 0, f"Negative percentile exists for {col}"
+        assert (
+            tiles_df[col].max() <= 1
+        ), f"Percentile over 100th exists for {col}"
+        assert (tiles_df[col].median() >= 0.4) & (
+            tiles_df[col].median() <= 0.6
+        ), f"Percentile distribution for {col} is decidedly not uniform"
+    return True
+
+
+def test_count_of_fips_codes(tiles_df, final_score_df):
+    final_score_state_count = (
+        final_score_df[field_names.GEOID_TRACT_FIELD].str[:2].nunique()
+    )
+    assert (
+        tiles_df["GTF"].str[:2].nunique() == final_score_state_count
+    ), "Some states are missing from tiles"
+    pfs_columns = tiles_df.filter(like="PFS").columns.to_list()
+    assert (
+        tiles_df.dropna(how="all", subset=pfs_columns)["GTF"].str[:2].nunique()
+        == 56
+    ), "Some states do not have any percentile data"
+
+
+def test_column_presence(tiles_df):
+    expected_column_names = set(TILES_SCORE_COLUMNS.values()) | {
+        THRESHOLD_COUNT_TO_SHOW_FIELD_NAME,
```

Comment:
i think we are going to cut these two fields out (at least UI Experience, for sure)